app/src/main/res/layout/fragment_geraete_edit.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:fillViewport="true"
+    tools:context =".ui.geraete.geraet_edit.GeraeteEditVerbraucherFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+
+
+        <TextView
+            android:id="@+id/geraet_edit_TextName"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/geraet_edit_textview_margin"
+            android:layout_marginTop="16dp"
```

Comment:
Ich glaub da haben wir auch schon was in dimen. Wöäre gut, wenn du das auch noch in Dimen auslagerst.

Reply:
Ja, die Auslagerung in dimen muss ich noch weiter machen bzw. sind wir da wahrscheinlich auch auf unterschiedlichen Versionen der dimens Klasse, deswegen habe ich das erstmal nach hinten verschoben.

Reply:
Ja das ist eine gute Sache. Muss ich ja auch noch machen